app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
You need to inherit from Player, ABC here, because if you are inheriting from Player only - you are required to implement  abstract methods here